src/lsc_user.c

```diff
@@ -43,7 +43,7 @@
  */
 #define G_LOG_DOMAIN "md manage"
 
-
```

Comment:
These are page breaks.  Useful in editors like Emacs to navigate between sections in a file.

Happy for them to be removed though, because I don't use them much and they confuse everyone.  And we should rather have smaller files.

Reply:
Ah, good to know. I have removed this from the PR, probably should be handled in a separate PR then.